packages/components/src/List/DisplayTile/DisplayTile.component.js

```diff
@@ -155,9 +156,10 @@ function DisplayTile(props) {
 		titleProps,
 	} = props;
 	const { width } = itemProps || {};
+	const hasItems = items && items.length !== 0;
```

Comment:
Test on items truthiness is not necesary as we have defaultProps.
So you can remove this const and test directly items.length in the render()